.github/labeler.yml

```diff
@@ -0,0 +1,18 @@
+documentation:
+  - documentation/**/*
+
+System tests:
+  - systemtest/**/*
+  - test/**/*
+
+cluster operator:
+  - cluster-operator/**/*
+
+topic operator:
+  - topic-operator/**/*
+
+user operator:
+  - user-operator/**/*
+
+docker images:
+  - docker-images/**/*
```

Comment:
I suppose we should have api in here too?

Reply:
I can add `api` as well if you want. But up until now we did not have `api` label at all so I'm not sure it is needed.

Reply:
I guess in the majority of cases something which touches api will touch on of the operators. The exception to that could be changes which touch `api` and only `operator-common`. Still I suspect this is good enough for now.

Reply:
Yeah, the way this works is that all matching labels are assigned. There will be almost no `api` only PRs. Most PRs changing the `api` will have it's part in one of the operators. We can always add more labels later.